_data/releases/2021-07/dotnet.yml

```
@@ -0,0 +1,299 @@
+
+    - Jason Dryhurst-Smith _([GitHub](https://github.com/jasond-s))_
+    - Oscar Cabrero _([GitHub](https://github.com/oscarcabrero))_
+
+    #### Fixed
+
+    - The retry policy used by clients will no longer overflow the `TimeSpan` maximum when using an `Exponential` strategy with a large number of retries and long delay set.
+
+    - The name of the property displayed in the `ArgumentOutOfRangeException` in the `MaxDeliveryCount` property in `SubscriptionProperties` was updated to use the correct property name.  (A community contribution, courtesy of _[oscarcabrero](https://github.com/oscarcabrero))_
+- Name: Microsoft.Azure.WebJobs.Extensions.ServiceBus
+  Version: 5.0.0-beta.4
+  DisplayName: WebJobs Extensions - Service Bus
+  ServiceName: WebJobs
+  VersionType: Beta
+  Hidden: false
+  ChangelogUrl: https://github.com/Azure/azure-sdk-for-net/tree/Microsoft.Azure.WebJobs.Extensions.ServiceBus_5.0.0-beta.4/sdk/servicebus/Microsoft.Azure.WebJobs.Extensions.ServiceBus/CHANGELOG.md
+  ChangelogContent: |-
+    #### Added
+    - Added `AutoCompleteMessages` property to `ServiceBusTriggerAttribute` which allows configuring autocompletion at the function level.
+
+    #### Key Bug Fixes
+    - Fix binding for DateTime parameters
+    - Avoid exception that occurred when a function settles messages and `AutoCompleteMessages` is true for multiple dispatch functions.
+    - Avoid null reference exception that could occur when no messages are available for a multiple dispatch function.
+- Name: Azure.Storage.Blobs
+  Version: 12.9.1
+  DisplayName: Storage - Blobs
+  ServiceName: Storage
+  VersionType: Patch
+  Hidden: false
+  ChangelogUrl: https://github.com/Azure/azure-sdk-for-net/tree/Azure.Storage.Blobs_12.9.1/sdk/storage/Azure.Storage.Blobs/CHANGELOG.md
+  ChangelogContent: '- Added optimization to unwrap encryption key once for DownloadTo and OpenRead when Client Side Encryption is enabled.'
+- Name: Azure.Core
+  Version: 1.16.0
+  DisplayName: Core
+  ServiceName: Core
+  VersionType: GA
+  Hidden: false
+  ChangelogUrl: https://github.com/Azure/azure-sdk-for-net/tree/Azure.Core_1.16.0/sdk/core/Azure.Core/CHANGELOG.md
+  ChangelogContent: |-
+    #### Changed
+
+    - Added `TenantId` to the properties on `TokenRequestContext` to enable multi-tenant support in Azure.Identity.
+- Name: Azure.Core.Amqp
+  Version: 1.2.0
+  DisplayName: Core - AMQP
+  ServiceName: Core
+  VersionType: GA
+  Hidden: false
+  ChangelogUrl: https://github.com/Azure/azure-sdk-for-net/tree/Azure.Core.Amqp_1.2.0/sdk/core/Azure.Core.Amqp/CHANGELOG.md
+  ChangelogContent: |-
+    #### Added
+    - All section properties of the `AmqpAnnotatedMessage` are now lazily allocated to reflect that they are defined as optional in the AMQP specification, section 3.2.
+
+    - The `HasSection` method has been added to `AmqpAnnotatedMessage` to allow inspecting the property for a section to determine if it is populated without triggering an allocation.
```

```suggestion
    #### Added
    - General availability for Sequence and Value body messages.
    
    - All section properties of the `AmqpAnnotatedMessage` are now lazily allocated to reflect that they are defined as optional in the AMQP specification, section 3.2.

    - The `HasSection` method has been added to `AmqpAnnotatedMessage` to allow inspecting the property for a section to determine if it is populated without triggering an allocation.
```

Including changes from the 1.1.0 release as well

---

Why include changes from previous release?

---

This was included in June release notes, so we can leave it off of July.